files/homeassistant/hacs_version/individual_control.yaml

```diff
@@ -262,16 +261,18 @@ cards:
     cards:
       - type: entities
         entities:
-          - entity: sensor.{HA_PRINTER_DEVICE_NAME}_current_stage
-            name: Stage
-          - entity: binary_sensor.{HA_PRINTER_DEVICE_NAME}_hms_errors
-            name: HMS Notifications
+          - entity: sensor.{HA_PRINTER_DEVICE_NAME}_remaining_time
+            name: Remaining Time
           - entity: sensor.{HA_PRINTER_DEVICE_NAME}_start_time
             name: Start Time
             secondary_info: last-changed
             attribute: start_time
           - entity: sensor.{HA_PRINTER_DEVICE_NAME}_end_time
             name: End Time
             attribute: end_time
-          - entity: sensor.{HA_PRINTER_DEVICE_NAME}_remaining_time
-            name: Remaining Time
+          - entity: sensor.{HA_PRINTER_DEVICE_NAME}_current_stage
+            name: Stage
```

Comment:
Any reason to swap remaining time and stage order? Idea before was "what is printer doing, then if it was printing, check when it started, ends, how much left". Moving time remaining to the top and stage to the middle means whenever the printer is idle, there's unneeded information up top before it.

Reply:
Now that you said it, it makes a lot more sense that way.
I'll restore it.